.travis.yml

```diff
@@ -1,6 +1,16 @@
+sudo: required
+
 language: ruby
 rvm:
   - 2.0
   - 2.1
   - 2.2
   - 2.3.0
+env:
+  - RUN_ON_LIVE_SERVER=1 VCR_UPDATE=1 SKIP_SSL_WITHOUT_CERTIFICATE_TEST=1 \
```

Comment:
Only cosmetic (and no requirement to change): I would find one per line more easy to quickly see what is set.